docs/strategy-advanced.md

```diff
@@ -197,27 +205,29 @@ class AwesomeStrategy(IStrategy):
                         current_rate: float, current_profit: float, **kwargs) -> float:
 
         if current_profit < 0.04:
-            return -1 # return a value bigger than the inital stoploss to keep using the inital stoploss
+            return 1 # return a value bigger than the inital stoploss to keep using the inital stoploss
 
         # After reaching the desired offset, allow the stoploss to trail by half the profit
-        desired_stoploss = current_profit / 2
-
         # Use a minimum of 2.5% and a maximum of 5%
-        return max(min(desired_stoploss, 0.05), 0.025)
+        desired_stop_from_open = max(min(current_profit / 2, 0.05), 0.025)
+
+        return stoploss_from_open(desired_stop_from_open, current_profit)
```

Comment:
I am not certain if applying `stoploss_from_open()` in this example is correct.

> Use the initial stoploss until the profit is above 4%, then use a trailing stoploss of 50% of the current profit with a minimum of 2.5% and a maximum of 5%.

The way i understand this example:
* profit < 4%  - use initial "static" stoploss.
* profit > 4%
  * 4.2% - trail with 2.1% (so stoploss should be set 2.1% below the current stoploss)
  * 5% - trail with 2.5% 
  * 20% - trail with 10% (50% of the profits - but capped to 5% by the above min/max limits) 
     This is where i think the "from open" goes wrong. Up to this point, the relative stoploss (actually accidentally, by using "half" the current profit) was always at the mid-point between profit and open, so both will return roughly the same result (not exactly - but close enough not to notice).
     The correct return for this case however should still be 5% - as I'd expect ~15% of the profits to be locked in - and trail with 5% behind my 20% profit (which obviously, i hope continues to 200%). By using `stoploss_from_open()`, the stoploss would be set to 5% above open (and never moved again as it's "top-capped" by the min/max limits) - potentially costing me most of the profits.

Reply:
Ah, yes you're right, in order to keep the 5% trailing, it would need to be:

```
        desired_stop_from_open = current_profit - max(min(current_profit / 2, 0.05), 0.025)
```

But at this point I think for the sake of simplicity it's probably better just to revert this example to the way it was before.  Either way works, but the original is definitely simpler, and in this case doesn't change the result very much.  `stoploss_from_open()` here is probably more confusing than helpful.